cmd/edit/ingress/cmd.go

```diff
@@ -284,30 +284,22 @@ func run(cmd *cobra.Command, argv []string) {
 			r.Reporter.Errorf("Updating Load Balancer Type is not supported for Hosted Control Plane clusters")
 			os.Exit(1)
 		}
-		if ocm.IsSts(cluster) {
-			r.Reporter.Errorf("Updating Load Balancer Type is not supported for STS clusters")
-			os.Exit(1)
-		}
 		lbType = &args.lbType
-	} else {
-		if interactive.Enabled() {
-			if !ocm.IsSts(cluster) {
```

Comment:
With this check removed, do we need to check `!ocm.IsHyperShiftCluster(cluster)` before prompting the message?

Reply:
Oh true

Reply:
Fixed, thanks!